ember-cli-build.js

```diff
@@ -29,6 +29,11 @@ module.exports = function(defaults) {
     app.import(app.bowerDirectory + '/mousetrap/mousetrap.js');
     app.import(app.bowerDirectory + '/normalize-css/normalize.css');
     app.import(app.bowerDirectory + '/oboe/dist/oboe-browser.js');
+    app.import(app.bowerDirectory + '/js-cookie/src/js.cookie.js', {
+        using: [
+            { transformation: 'amd', as: 'js-cookie' }
+        ],
+    });
```

Comment:
@johnkord if this is the right approach, I think we should definitely add a comment here explaining why we're importing js-cookie. Without knowledge of sysdig-inspect-web, there's no way to know why this is needed.

Reply:
Ideally, we should find a way to remove the dependency entirely. Otherwise it's a good point.